### DISCUSSION TITLE
Group Assignment #1

### DISCUSSION THREAD

**Discussion by yohman** [yohman/21S-DH151#21] [Assignments]
Read the [instructions](https://github.com/yohman/21S-DH151/blob/main/Group%20Assignments/GroupAssignment1.md) carefully.

Submit the following:
- Your Group name
- Your Group Repo URL
- Group members
- Link to your group proposal

**Comment by kimjee8955** [comment]
Group Name: Datart
Group Repo: https://github.com/kimjee8955/DH151-Datart
Members: Annmarie Cong, Eustina Kim, Michelle Lee
Link to proposal: https://github.com/kimjee8955/DH151-Datart/blob/main/GroupAssignment1.md

**Reply by yohman** [reply]
This is a great start to your project, and I especially like how you have thoughtfully looked for _usable_ open data for your map. Some of the art data is good to go as-is, while others may need some cleanup/geocoding. Combined, they will form a rather disparate collection of art in Los Angeles, so it will be a challenge to inform users as to exactly what is being shown and what is not. Will you find and add more art data? Might you create some data of you own? 

Begin to think as to what the narrative (beyond the points on the map) will be. How can you "thicken" the map with a more immersive user experience? What additional layers can you add that will add value? Filters? Interactive components? Metrics? I look forward to your ideas moving forward.

**Comment by justisg** [comment]
Group Name: The Musical Maps
Group Repo: https://github.com/justisg/TheMusicalMaps
Members: Kai Watanabe, Justis Guardia, Daniel Liu
Proposal: https://github.com/justisg/TheMusicalMaps#readme

**Reply by yohman** [reply]
Your group seems well equipped to take on a high level of computational methods. There is a delicate balance between a subjective taste of music, trends, temporality, and what the pandemic toll has to do with all of this. 

As one possible narrative flow, the artist (songkick) API may provide a geospatial journey per artist, and it may allow for an immersive, non-static interaction by the user. Which artists were active before/after the pandemic? Can you add to this additional "thick" layers of concert venues, events, and twitter and spotify analytics without convoluting the interface and user experience?

I am excited about the possibilities here, but know that the data challenge is real. A good understanding of how your various API endpoints work, and how your interface can extract the information in a way that tells a compelling narrative will be the next challenge.

**Comment by ivanma9** [comment]
Group Name: Happy Tweet
Group Repo: https://github.com/ivanma9/HappyTweet
Members: MJ Howland, Ivan Ma, Josephine Meyer-Hogan
Proposal: https://github.com/ivanma9/HappyTweet/blob/main/Group%20Assignments/GroupAssignment1.md

**Reply by yohman** [reply]
Are world happiness rankings and twitter conversations correlated? Essentially, this is what your group is attempting to visualize. The global happiness data is pretty straight forward, and should provide a view into global rankings of happiness over the past 5 years. The bigger challenge will be in incorporating twitter data to your project. The Twitter API is not easy to interact with, and assuming that happiness will be measured through some sort of sentiment analysis, a thorough understanding and methodology with Twitter data needs to be laid out in order to make this work.

I also look forward to hearing what the group intends to do in terms of the user interface design. How will a user learn from your site? What is the narrative flow? Will there be filters and interactive elements that allow users to say, navigate to certain countries to dig deeper? Are there additional "thick" layers that may be informative?

**Comment by lisa58** [comment]
Group Name: Fantastic Four
Group Repo: https://github.com/LynnT2/FantasticFour
Members: Labiba Alam, Haiqi Zhou, Adriana Romero, and Lynn Tieu 
Proposal: https://github.com/LynnT2/FantasticFour/blob/main/GroupAssignment1.md

**Reply by yohman** [reply]
This is such a relevant and important issue, and I commend the group for taking on the challenge. This will be a unique project in a sense that you will have community based support (assuming you work with MovementHub), which makes it even more compelling and meaningful due to the academic/community collaboration. In a sense, the project will be structured as a "client project," working closely with your community partner, assessing their needs with what is doable within the scope and timeframe of the course. In short, along with the technical development, project management will be an urgent need here.

As you may have figured out by now, "hate crime" data is difficult to obtain, given the sensitivity and objectiveness of what is considered "hate" in racial terms. Leverage on the knowledge that MovementHub has already journeyed on, and come up with a data intake methodology that is doable. Then, think carefully about how the site will function to expose such "hate" triggered incidents, and how it will help to inform, expose, and add urgency to this issue. Keep it simple, but powerful.

**Comment by hoz-map** [comment]
Group Name: Rethink Recycling
Group Repo: https://github.com/hoz-map/Rethink-Recycling
Members: Alice Lu, Emily Chau, and Hozumi Oshida
Proposal: https://github.com/hoz-map/Rethink-Recycling#readme

**Reply by yohman** [reply]
I like how your project overview _already_ tells a story. It is this journey of recycled waste that is at the heart of your project, and how well you can have users immerse themselves geospatially through your mapping interface is the challenge. Your data suggests a global beginning, with solid metrics driven from the Science Journal article, and leveraged with World Bank data. The Hong Kong data could allow for an in depth look at a local situation as one of the world's largest waste importers. 

A very thoughtful and promising start, and I look forward to the groups ideas on how the user interface design will formulate.

**Comment by JuliaWood1** [comment]
Group Name: Ocean Pollution Posse
Group Repo: https://github.com/eliaranita1/project1
Members: Julia Wood, Eli Aranita, Derek Wang
Proposal: https://github.com/eliaranita1/project1/blob/main/ProjectProposal.md

**Reply by yohman** [reply]
Hello Posse! I too, am surprised to learn that fishing gear is the top contributor to ocean plastic waste. Who would have thought? Which makes this project that much more compelling and informative. How you go about it will be the challenge moving forward.

The plastic data set is a great start to reveal the problem as a global issue. Without seeing the fishing dataset, your description does appear to reveal a very rich and unique geotemporal layer which could allow for a very compelling interactive narrative. That said, I wonder what the team is thinking in terms of not duplicating the efforts that have already been done by the Global Fishing Watch folks here: https://globalfishingwatch.org/map/? How else could you "thicken" the spatial narrative?

**Comment by ryanorlando** [comment]
Group Name: Team Data
Group Repo: https://github.com/ryanorlando/DH151-Project
Members: Ainsleigh Douglas, Sebastien Dills, Ryan Orlando
Proposal: https://github.com/ryanorlando/DH151-Project/blob/main/Assignment1.md

**Reply by yohman** [reply]
This project has tremendous promise in whether or not the "dark" side of technology innovations are shouldered by vulnerable populations around the world. Is our day-to-day reliance on technology inadvertently making "invisible" populations suffer?

The proposed datasets can provide a starting point to your investigation. You may choose to create a choropleth map based on world bank indicators, and on top of that map the data center locations. But how you "thicken" the map beyond the points and colors will be the challenge. Can additional layers add to the narrative? What narrative are you informing? What interactions will make the site immersive and revealing of the problem you stipulate? I look forward to seeing what the group is thinking for the user interface design.

**Comment by lisalou36** [comment]
Group Name:  Environmental Racism in Carceral California
Group Repo: https://github.com/jmtanenbaum/DHEnviroIncarceration
Members: Hannah Lien, Julia Tanenbaum, Lisa Kahn
Proposal: https://github.com/jmtanenbaum/DHEnviroIncarceration/blob/main/README.md

**Reply by yohman** [reply]
This is a very well written, thoughtful proposal that brings the urgency of the "problem" to light, and implores of the need for attention. Given its scholarly approach, consider adding resources (even within the proposal) with bibliographical entries at the end. 

The methodology and technical scope do not give light as to how you intend to develop the site, i.e. how will each layer of data be analyzed/visualized. The data section needs to includes links to the data pages themselves for me to evaluate their usability.

**Comment by frida-barragan** [comment]
Group name: The Health Mappers
Group repo: https://github.com/frida-barragan/DH151_Group
Members: Frida Barragan & Kate McInerny
Proposal: https://github.com/frida-barragan/DH151_Group/blob/main/Proposal.md

**Reply by yohman** [reply]
I applaud the group's effort to address and illuminate an important health crisis in our county. The group should start to develop ideas on exactly how the data layers will be incorporated in an interactive web map, and how the user will navigate and use the site effectively. How will your narrative of mental health disparities be exposed on the interactive map? The data from DPH is summarized at different levels of geography (places, SPA's) and may need added attention (data cleanup/exploration) to make sure it can be used within the scope of the project.

**Comment by kaicolorado** [comment]
Group Name: cARTographers
Group Repo: https://github.com/kaicolorado/DH151-Group
Members: Kai Colorado & Alex Haddad
Proposal: https://github.com/kaicolorado/DH151-Group/blob/main/Proposal.md

**Reply by yohman** [reply]
A good start for an interesting perspective on academic policies. The data layers are at the state level, and the nation's report card maproom is a good start to understand what is "doable" with this type of data. Overlaying state level measures on top of one another may not be enough to illuminate the narrative you are trying to expose. Consider additional metrics or qualitative layers to "thicken" your narrative beyond simple layer toggling.

**Comment by alekhyavittalam** [comment]
Group Name: State-of-the-Arts
Group Repo: https://github.com/alekhyavittalam/state-of-the-arts
Members: Natasha Grace, Zona Liao and Alekhya Vittalam
Proposal: https://github.com/alekhyavittalam/state-of-the-arts/blob/main/Assignment1.md

**Reply by yohman** [reply]
Your methodology outlines the narrative you intend to have your map illustrate in a clear and concise manner. I wonder, however, if the datasets you provide will allow you to do so. Consider a more thorough data exploration, including world indicators on poverty and health rankings, and other layers that may aid in "thickening" the map. A qualitative narrative flow (including historical/journalistic imagery and video perhaps?) may also help to strengthen the website beyond the map with quantitative layers.
